.github/workflows/prepare-staging.yml

```
@@ -0,0 +1,49 @@
+
+permissions:
+  contents: write
+
+concurrency:
+  group: prepare-staging
+  cancel-in-progress: false
+
+jobs:
+  prepare:
+    # The prepared marker commit re-triggers this workflow; do not prepare twice.
+    if: ${{ !startsWith(github.event.head_commit.message, 'chore(staging): prepared') }}
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)